os.bpmsuite.businesscentral/added/launch/bpmsuite-businesscentral.sh

```diff
@@ -119,3 +127,33 @@ function configure_misc_security() {
 function configure_metaspace() {
     export GC_MAX_METASPACE_SIZE=${WORKBENCH_MAX_METASPACE_SIZE:-1024}
 }
+
+# required envs for HA
+function haEnabled() {
+  [ -n "$APPFORMER_ELASTIC_HOST" -a -n "$APPFORMER_JMS_BROKER_USER" -a -n "$APPFORMER_JMS_BROKER_PASSWORD" -a -n "$APPFORMER_JMS_BROKER_ADDRESS" ]
+}
+
+function configureHA() {
```

Comment:
Can this function be renamed to configure_ha? All our other functions use under_scores vs. camelCase. So, this would be for consistency.